gadgets/trace_exec/gadget.yaml

```diff
@@ -3,87 +3,78 @@ description: trace process executions
 homepageURL: https://inspektor-gadget.io/
 documentationURL: https://inspektor-gadget.io/docs
 sourceURL: https://github.com/inspektor-gadget/inspektor-gadget/
+datasources:
+  exec:
+    fields:
+      timestamp_raw:
+        annotations:
+          columns.hidden: true
+      timestamp:
+        annotations:
+          template: timestamp
+      pid:
+        annotations:
+          template: pid
+      ppid:
+        annotations:
+          template: pid
+      comm:
+        annotations:
+          template: comm
+      uid:
+        annotations:
+          template: uid
+          columns.hidden: true
+      gid:
+        annotations:
+          template: uid
+          columns.hidden: true
```

Comment:
Weren't hidden before

Reply:
I tuned it because this gadget is printing too many things. (In any case, I think we'll need to make a pass later on tuning all gadgets)